### PR TITLE
Updated to GitHub Actions

### PR DESCRIPTION
**Pull request by eslutz**
## Description

Removed AppVeyor config and added GitHub Actions workflows.

## Type of change: 1

1. Non-functional change (e.g. documentation changes, removing unused `using` directives, renaming local variables, etc)
2. Bug fix (non-breaking change that fixes an issue)
3. New feature (non-breaking change that adds functionality)
4. Breaking change (fix or feature that could cause existing functionality to not work as expected)

## Checklist:

- Have you reviewed your own code? Do you understand every change?
- Are you following the [contributing guidelines](../blob/main/CONTRIBUTING.md)?
- Have you added tests that prove your fix is effective or that this feature works?
- New and existing unit tests pass locally with these changes?
- Have you made corresponding changes to the documentation?
- Will this change require an update to an example project? (if so, create an issue and link to it)

---

<sup>_[Reviewer guidelines](../blob/main/CONTRIBUTING.md#reviewing-changes)_</sup>

### CODE REVIEW

**Review by NeilSchiefer — APPROVED** [review]
Sure, you complete this *after* I manually make a release this morning.:)

**Comment by eslutz** [comment]
> Sure, you complete this _after_ I manually make a release this morning.:)

Hey, at least it will be ready for when you make a full release :laughing:

File: RockLib.DistributedTracing.AspNetCore/RockLib.DistributedTracing.AspNetCore.csproj [hunk 1]
```diff
@@ -23,13 +23,17 @@
...
+	  <None Remove="System.Text.Encodings.Web" />
+	</ItemGroup>
 	<ItemGroup>
 		<None Include="..\LICENSE.md" Pack="true" PackagePath="" />
 		<None Include="..\icon.png" Pack="true" PackagePath="" />
 	</ItemGroup>
 	<ItemGroup>
 		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
 		<PackageReference Include="OpenTelemetry" Version="1.3.1" />
+		<PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" />
```
Comment (NeilSchiefer) [inline RockLib.DistributedTracing.AspNetCore/RockLib.DistributedTracing.AspNetCore.csproj:36]: This was from the Http.Abstractions could you move it down to there?

Comment (eslutz) [reply RockLib.DistributedTracing.AspNetCore/RockLib.DistributedTracing.AspNetCore.csproj]: Moved the package reference location.

File: RockLib.DistributedTracing.AspNetCore/RockLib.DistributedTracing.AspNetCore.csproj [hunk 2]
```diff
@@ -23,6 +23,9 @@
 	<PropertyGroup Condition="'$(Configuration)'=='Release'">
 		<ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
 	</PropertyGroup>
+	<ItemGroup>
```
Comment (NeilSchiefer) [inline RockLib.DistributedTracing.AspNetCore/RockLib.DistributedTracing.AspNetCore.csproj:26]: could remove this now then I think its good.

**Pull request merged**